core/tests/protractor_desktop/topicAndStoryEditor.js

```diff
@@ -296,9 +284,9 @@ describe('Chapter editor functionality', function() {
   });
 
   it('should create a basic chapter.', async function() {
-    await storyEditorPage.createNewChapter('Chapter 1');
+    await storyEditorPage.createNewChapter(
+      'Chapter 1', dummyExplorationIds[0], '../data/test_svg.svg');
```

Comment:
Since we reuse the `test_svg.svg` path, can we make it a constant?

Reply:
Moved it to constants file. Done